gui/launcher.lua

```diff
@@ -258,6 +269,34 @@ function EditPanel:init()
             on_submit2=function()
                 self.on_submit2(self.subviews.editfield.text) end},
     }
+    self:update_pause_label(false,true)
+end
+
+function EditPanel:pause_fort()
+    if dfhack.isMapLoaded() then
+        df.global.pause_state = not df.global.pause_state
+    end
+    self:update_pause_label(false)
+end
+function EditPanel:enable_pause_label()
```

Comment:
instead of changing the colors, how about changing the hotkey hint from "Space" to "Double Space"? That might make the intent a little clearer.

Reply:
ehhhhhh i don't know if "Double Space" would necessarily make it "clearer". It would be descriptive, but doesn't necessarily explain _why_ the user has to press spacebar twice for it to unpause.

the real reason is, like, because after the first space, the second is passed to the game or something? There _has_ to be a better way to convey this